spec/lib/api/v3/work_packages/schema/work_package_schema_spec.rb

```diff
@@ -92,9 +79,22 @@
           expect(subject.assignable_statuses_for(user)).to eql(status_result)
         end
       end
+    end
+
+    describe '#available_custom_fields' do
+      let(:cf1) { double }
+      let(:cf2) { double }
+      let(:cf3) { double }
+
+      before do
+        scope_double = double
+        allow(scope_double).to receive(:all).and_return([cf1, cf2])
+        allow(type).to receive(:custom_fields).and_return(scope_double)
+        allow(project).to receive(:all_work_package_custom_fields).and_return([cf2, cf3])
+      end
 
-      describe '#available_custom_fields' do
-        it_behaves_like 'WorkPackageSchema#available_custom_fields'
+      it 'is expected to return custom fields available in project AND type' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.